sdk/search/Azure.Search.Documents/tests/DocumentOperations/IndexingTests.cs

```diff
@@ -349,6 +348,92 @@ public async Task StaticDocuments()
+                        })
+#endif
+                });
+            UncasedHotel expected = new UncasedHotel
+            {
+                HotelId = "1",
+                HotelName = "Secret Point Motel",
+                Description = "The hotel is ideally located on the main commercial artery of the city in the heart of New York. A few minutes away is Time's Square and the historic centre of the city, as well as other places of interest that make New York one of America's most attractive and cosmopolitan cities.",
+                DescriptionFr = "L'hôtel est idéalement situé sur la principale artère commerciale de la ville en plein cœur de New York. A quelques minutes se trouve la place du temps et le centre historique de la ville, ainsi que d'autres lieux d'intérêt qui font de New York l'une des villes les plus attractives et cosmopolites de l'Amérique.",
+                Category = "Boutique",
+                Tags = new[] { "pool", "air conditioning", "concierge" },
+                ParkingIncluded = false,
+                SmokingAllowed = true,
+                LastRenovationDate = new DateTimeOffset(1970, 1, 18, 0, 0, 0, TimeSpan.FromHours(-5)),
+                Rating = 4,
+                Location = TestExtensions.CreatePoint(-73.975403, 40.760586),
+                Address = new HotelAddress
+                {
+                    StreetAddress = "677 5th Ave",
+                    City = "New York",
+                    StateProvince = "NY",
+                    Country = "USA",
+                    PostalCode = "10022"
+                },
+                Rooms = new[]
+                {
+                    new HotelRoom
+                    {
+                        Description = "Budget Room, 1 Queen Bed (Cityside)",
+                        DescriptionFr = "Chambre Économique, 1 grand lit (côté ville)",
+                        Type = "Budget Room",
+                        BaseRate = 9.69,
+                        BedOptions = "1 Queen Bed",
+                        SleepsCount = 2,
+                        SmokingAllowed = true,
+                        Tags = new[] { "vcr/dvd" }
+                    },
+                    new HotelRoom
+                    {
+                        Description = "Budget Room, 1 King Bed (Mountain View)",
+                        DescriptionFr = "Chambre Économique, 1 très grand lit (Mountain View)",
+                        Type = "Budget Room",
+                        BaseRate = 8.09,
+                        BedOptions = "1 King Bed",
+                        SleepsCount = 2,
+                        SmokingAllowed = true,
+                        Tags = new[] { "vcr/dvd", "jacuzzi tub" }
+                    }
+                }
+            };
+            IndexDocumentsBatch<UncasedHotel> batch = IndexDocumentsBatch.Create(
+                IndexDocumentsAction.Upload(expected));
+
+            Response<IndexDocumentsResult> response = await client.IndexDocumentsAsync(batch);
```

Comment:
You don't need to wait in between some of these calls like in other places? 🤔

Reply:
The first bit creates the batch, then we send it and tear apart the result.  We add a wait a couple lines down with `await resources.WaitForIndexingAsync();`.